README.md

```diff
@@ -583,6 +584,7 @@ Table of Contents
   * [getstream.io](https://getstream.io/) — Build scalable news feeds and activity streams in a few hours instead of weeks, free for 3 million feed updates/month
   * [hasura.io](https://hasura.io/) — Platform to build and deploy app backends fast, free for single node cluster.
   * [iron.io](https://www.iron.io/) — Async task processing (like AWS Lambda) with free tier and 1-month free trial
+  * [linx.software](https://linx.software/) — Visual, low-code IDE to easily develop the backend for websites or mobile apps. 14 day free hosting.
```

Comment:
Unfortunately this doesnt fit with the list then, quoting the top of the file:

```
NOTE: This list is only for as-a-Service offerings, not for self-hosted software. For a service to be eligible it has to offer a Free Tier and not just a free trial. If the Free Tier is time bucketed it has to be at least a year.
```

Reply:
The Linx IDE is free forever. I will change the wording again to reflect that. 
What about the Open API Designer? Is that OK?
Thanks

Reply:
api designer good, the IDE would need to be removed if as I understand its kind of tied to the online service

Reply:
OK, I removed the IDE

Reply:
thank you, sux but so it is, the whole usable unit has to have a long term free tier